### ISSUE TITLE
[Feature Request] Reputation point fix and Disable rubberband

### ISSUE THREAD

**Issue by MeekGroundhog21** [ExOptsTeam/NFSU2ExOpts#45]
There is a requirement of reputation points to get all the sponsors for all 5 sponsors so pls add a feature to either allow us to remove the reputation points or lower the required reputation points to get all 5 sponsors

In races, you have to be a certain amount ahead to get 300 reputation points and they are absurd so add a feature that fixes this issue in the circuit, street x, sprint, and URL you have to be ahead by a set amount of seconds to get 300 points

if possible add a feature to disable rubberbanding

**Comment by GGuyX** [comment]
I don't think this is still worked on sadly. Or at least I hope the project wasn't abandoned but we don't really know anything until there's some announcement or anything like that.
